### ISSUE TITLE
ERROR Plugin load failed: hexo-renderer-sass

### ISSUE THREAD

**Issue by kikitaMoon** [kikitaMoon/kikitamap#9]
1. Git Bash Here in `HexoBlogSource` folder
2. `$ hexo server`
    ```Bash
    ERROR Plugin load failed: hexo-renderer-sass
    Error: Node Sass does not yet support your current environment: Windows 64-bit with Unsupported runtime (108)
    For more information on which environments are supported please see:
    https://github.com/sass/node-sass/releases/tag/v4.14.1
        at module.exports (C:\MykikitaMap\BlogSource\HexoBlogSource\node_modules\node-sass\lib\binding.js:13:13)
        at Object.<anonymous> (C:\MykikitaMap\BlogSource\HexoBlogSource\node_modules\node-sass\lib\index.js:14:35)
        at Module._compile (node:internal/modules/cjs/loader:1254:14)
        at Module._extensions..js (node:internal/modules/cjs/loader:1308:10)
        at Module.load (node:internal/modules/cjs/loader:1117:32)
        at Module._load (node:internal/modules/cjs/loader:958:12)
        at Module.require (node:internal/modules/cjs/loader:1141:19)
        at require (node:internal/modules/cjs/helpers:110:18)
        at Object.<anonymous> (C:\MykikitaMap\BlogSource\HexoBlogSource\node_modules\hexo-renderer-sass\lib\renderer.js:4:12)
        at Module._compile (node:internal/modules/cjs/loader:1254:14)
        at Module._extensions..js (node:internal/modules/cjs/loader:1308:10)
        at Module.load (node:internal/modules/cjs/loader:1117:32)
        at Module._load (node:internal/modules/cjs/loader:958:12)
        at Module.require (node:internal/modules/cjs/loader:1141:19)
        at require (C:\MykikitaMap\BlogSource\HexoBlogSource\node_modules\hexo\lib\hexo\index.js:216:21)
        at C:\MykikitaMap\BlogSource\HexoBlogSource\node_modules\hexo-renderer-sass\index.js:4:20
        at C:\MykikitaMap\BlogSource\HexoBlogSource\node_modules\hexo\lib\hexo\index.js:232:12
        at tryCatcher (C:\MykikitaMap\BlogSource\HexoBlogSource\node_modules\bluebird\js\release\util.js:16:23)
        at Promise._settlePromiseFromHandler (C:\MykikitaMap\BlogSource\HexoBlogSource\node_modules\bluebird\js\release\promise.js:512:31)
        at Promise._settlePromise (C:\MykikitaMap\BlogSource\HexoBlogSource\node_modules\bluebird\js\release\promise.js:569:18)
        at Promise._settlePromise0 (C:\MykikitaMap\BlogSource\HexoBlogSource\node_modules\bluebird\js\release\promise.js:614:10)
        at Promise._settlePromises (C:\MykikitaMap\BlogSource\HexoBlogSource\node_modules\bluebird\js\release\promise.js:693:18)
    INFO  Start processing
    INFO  Hexo is running at http://localhost:4000/. Press Ctrl+C to stop.
    INFO  See you again
    ```
3. The website can be accessed via `http://localhost:4000`.

**Comment by kikitaMoon** [comment]
### **Analysis:**

The error indicates that the `node-sass` library used by the `hexo-renderer-sass` plugin is incompatible with your current Node.js version. The `node-sass` library is deprecated and often problematic with newer Node.js versions. The recommended approach is to use **`sass` (Dart Sass)**.

### **Solution:** 

The `node-sass` library is deprecated and often problematic with newer Node.js versions. The recommended approach is to use **`sass` (Dart Sass)**.

1. **Uninstall `hexo-renderer-sass`**:
   
   ```Bash
   npm uninstall hexo-renderer-sass
   ```
   ```Bash
    npm WARN old lockfile
    npm WARN old lockfile The package-lock.json file was created with an old version of npm,
    npm WARN old lockfile so supplemental metadata must be fetched from the registry.
    npm WARN old lockfile
    npm WARN old lockfile This is a one-time fix-up, please be patient...
    npm WARN old lockfile
    npm WARN deprecated rimraf@2.4.5: Rimraf versions prior to v4 are no longer supported
    npm WARN deprecated titlecase@1.1.2: no longer maintained
    npm WARN deprecated uuid@3.2.1: Please upgrade  to version 7 or higher.  Older versions may use Math.random() in certain circumstances, which is known to be problematic.  See https://v8.dev/blog/math-random for details.
    npm WARN deprecated mkdirp@0.5.1: Legacy versions of mkdirp are no longer supported. Please update to mkdirp 1.x. (Note that the API surface has changed to use Promises in 1.x.)
    npm WARN deprecated hexo-bunyan@1.0.0: Please see https://github.com/hexojs/hexo-bunyan/issues/17
    npm WARN deprecated swig-templates@2.0.2: unmaintained

    removed 143 packages, changed 34 packages, and audited 629 packages in 21m
    
    2 packages are looking for funding
      run `npm fund` for details
    
    70 vulnerabilities (7 low, 16 moderate, 32 high, 15 critical)
    
    To address issues that do not require attention, run:
      npm audit fix
    
    To address all issues (including breaking changes), run:
      npm audit fix --force
    
    Run `npm audit` for details.
    npm notice
    npm notice New major version of npm available! 9.5.1 -> 11.0.0
    npm notice Changelog: <https://github.com/npm/cli/releases/tag/v11.0.0>
    npm notice Run `npm install -g npm@11.0.0` to update!
    npm notice
```

2.  Upgrade `Node.js` and `npm `to the latest.
 
      ```Bash
      $ node -v
      v22.12.0
      
      $ npm -v
      10.9.0
      
      $ npm install -g npm@11.0.0
      added 1 package in 14s
      24 packages are looking for funding
        run `npm fund` for details
      ````
     🔴  I just ignored many findings via `npm audit` and `npm audit fix` .


3. **Install `hexo-renderer-scss`** (which uses Dart Sass):
   ```bash
   npm install hexo-renderer-scss --save
   ```
   > **Note**: `hexo-renderer-scss` uses Dart Sass, which is more up-to-date and compatible with recent Node.js versions.


😮‍💨 There are too many packages out of date or deprecated without well maintaining. 
Change a new theme ....

**Comment by kikitaMoon** [comment]
Closing it and turn to a new theme.

Thanks and bye - Theme `livemylife`  - loved it.
https://github.com/V-Vincen/hexo-theme-livemylife
